Makefile

```diff
@@ -25,6 +25,12 @@ run_at_gcp_python:
 run_at_gcp_java: ${JAR}
 	gcp-utils/run-java-residue-count.sh M Q P R S
 
+.PHONY: run_at_gcp_java_blastdb
+run_at_gcp_java_blastdb: ${JAR}
+	gcp-utils/run-java-residue-count-blastdb.sh
```

Comment:
@boratyng it looks like gcp-utils/run-java-residue-count-blastdb.sh is missing. Could you please add it?

Reply:
I opened another pull request with the missing gcp-utils/run-java-residue-count-blastdb.sh.